marlowe-playground-client/src/Marlowe/Gen.purs

```diff
@@ -105,13 +105,13 @@ genBound = do
 genPosition :: forall m. MonadGen m => MonadRec m => m Pos
 genPosition = chooseInt 0 1000
 
-genRange :: forall m. MonadGen m => MonadRec m => m Range
+genRange :: forall m. MonadGen m => MonadRec m => m Location
 genRange = do
   startLineNumber <- genPosition
   startColumn <- genPosition
   endLineNumber <- genPosition
   endColumn <- genPosition
-  pure { startLineNumber, startColumn, endLineNumber, endColumn }
+  pure $ Range { startLineNumber, startColumn, endLineNumber, endColumn }
```

Comment:
another example of how we need to look at the tests and possible give the generators `MonadAsk Settings` where settings can define flags such as use Range or BlockId etc

Reply:
Not sure I follow, could you elaborate? 🧐

Reply:
actually I was wrong because I didn't notice that this function is specifically called `getRange`

Reply:
although, we still need more property tests